Gems/ROS2/Code/Source/RobotImporter/xacro/XacroUtils.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include "RobotImporter/URDF/UrdfParser.h"
+#include <AzCore/std/containers/unordered_map.h>
+#include <AzCore/std/string/string.h>
+
+namespace ROS2::Utils::xacro
+{
+
+    static const char* kXacroExecutable = "xacro";
+    using Params = AZStd::unordered_map<AZStd::string, AZStd::string>;
+    //! Structure that keeps all artifacts of xacro execution
+    struct ExecutionOutcome
+    {
+        //! parsed URDF from successful xacro's output
```

Comment:
uber nit: This types of comments in .h files tend to start with upper case characters (follow normal sentence rules) - so this just becomes `//! Parsed URDF from successful xacro's output.`